src/qrcode/schema.ts

```diff
@@ -42,6 +42,11 @@ export enum QRCodeDataType {
   Address = 'Address',
 }
 
+export enum QRCodeStyle {
+  New = 'New',
```

Comment:
feel like it's a slippery slope from this to NewModuleRedesigned. How about a more descriptive version name, like `Jan2023QRCodeStyle`? or `Jan2023Version`?